src/Gdi32.Desktop/Gdi32.cs

```diff
@@ -23,10 +23,30 @@ public static partial class Gdi32
         [DllImport(nameof(Gdi32), SetLastError = false)]
         public static extern int GetDeviceCaps(
             User32.SafeDCHandle hdc,
-            int nIndex);
+            DeviceCap nIndex);
```

Comment:
OK, I will. should we take the `int` overload out when the major change happens? If so I'll open an issue to be linked with the mark-2 version so that we can track all the needed changes that we encounter during the development process. May I?